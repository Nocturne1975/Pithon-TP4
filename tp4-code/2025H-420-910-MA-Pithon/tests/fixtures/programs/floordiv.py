# Test de la division entiere
a = 10
b = 3

c = a / b
print("Division normale: ", c)

d = a // b
print("Division entiere: ", d)

e = -10 // 3
print("Division entiere avec negatif: ", e)

f = a // 0
print("Division par zero: ", f)

