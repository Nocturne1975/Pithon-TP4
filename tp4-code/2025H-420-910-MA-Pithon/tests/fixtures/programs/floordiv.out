Division normale: 3.333333333335
Division entiere: 3
Division entiere avec negatif: -4
Division par zero: PithonZeroDivisionError
